Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -418,23 +446,38 @@ private void UpdatePosition()
             if (_animating)
                 return;
 
-            if (_settings.RememberLastLaunchLocation)
+            if (_settings.LauncherPosition == "RememberLastLaunchLocation")
```

Comment:
same switch for this

Reply:
I changed the part to call InitializePosition().